src/api.js

```diff
@@ -14,11 +14,13 @@ export const getTopics = () => {
 export const getArticles = (queries) => {
     const queryArr = []
     for (const key in queries){
-        queryArr.push(`?${key}=${queries.key}`)
+        if(queries[key] !== ""){
+            queryArr.push(`?${key}=${queries.key}`) 
+        }
     }
     const queryStr = queryArr.join("")
     return NCnewsAPI.get(`/articles${queryStr}`).then((response) => {
-        return response.data.articles
+        return response.data
```

Comment:
Totally valid way to deal with queries. Axios also has its own way to deal with queries by using an object as the second argument of .get
`NCnewsAPI.get("/articles", {params:{topic, order, sort_by})`